### PR TITLE
Blocks documentation

### PR DESCRIPTION
**Pull request by aduermael**
Adds documentation about `Block` + `Player.BlockUnderneath`

### CODE REVIEW

**Comment by aduermael** [comment]
I also improved the style a little.

**Comment by aduermael** [comment]
ping @gdevillele !

File: lua-docs/content/reference/Block.md [hunk 1]
```diff
@@ -5,4 +5,23 @@ keywords: particubes, game, mobile, scripting, cube, voxel, world
 
 # Block
 
+Usually represents one block in the map. But could also represent a block for any shape in the game.
+
 ## Fields
+
+### ID (Integer, read-only)
```
Comment (gdevillele) [inline lua-docs/content/reference/Block.md]: `ID` -> `Id` ?

**Comment by aduermael** [comment]
Block currently uses `ID`... But we should make it more consistent. 
It would be better to use `Id` everywhere.

**Comment by aduermael** [comment]
Merging it for now, I'll update the `Id` key once you're done with current Lua sandbox changes.

**Pull request merged**